### DISCUSSION TITLE
How can I get the name of appliedTags Id using discord.js

### DISCUSSION THREAD

**Discussion by JohnVersus** [discordjs/discord.js#9767] [Q&A] [answered]
I want to know how I can read the name of a tag in the forum thread channel. It seems like the data of the tag is stored as an Id in channel metadata. I could not find any way to get the name using this Id.

`appliedTags: [ '1139861707964035072' , '1139861707964074322' ]`

**Comment by sdanialraza** [comment, marked as answer]
You can use [ForumChannel#availableTags](https://old.discordjs.dev/#/docs/discord.js/main/class/ForumChannel?scrollTo=availableTags) which is an array of all the tags that can be used, you can use filter on it to get the tag matching the id you want:
```js
<ForumChannel>.availableTags.filter((tag) => tag.id === "tag_id")
```
